### PR TITLE
Fix watch test

### PR DESCRIPTION
**Pull request by alemagio**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by fox1t** [comment]
Ok it worked ` test/start.test.js 1> [fastify-cli] process forced end`. I think we can leave this active also on Actions.

**Review by fox1t — APPROVED** [review]
LGTM

File: lib/watch/fork.js [hunk 1]
```diff
@@ -20,9 +20,11 @@ process.on('message', function (event) {
...
     const message = chalk.red('[fastify-cli] process forced end')
     setTimeout(exit.bind({ message }), TIMEOUT).unref()
-    fastify.close(() => {
-      process.exit(0)
-    })
+    if (fastify) {
+      fastify.close(() => {
+        process.exit(0)
+      })
+    }
```
Comment (mcollina) [inline lib/watch/fork.js:27]: I think this is missing an `else` clause in which we exit immediately if `fastify` is not there. There is no point in waiting.

**Review by mcollina — CHANGES_REQUESTED** [review]
lgtm

File: lib/watch/fork.js (hunk 1 above)
Comment (fox1t) [reply lib/watch/fork.js:27]: Good point! In that case, we don't need a `GRACEFUL_SHUT` at all!

Comment (alemagio) [reply lib/watch/fork.js:27]: by adding
`else {
  process.exit(0)
}`
makes it exit without waiting for the message `[fastify-cli] process forced end` to appear.
Is that ok?

Comment (mcollina) [reply lib/watch/fork.js:27]: yes, the app is not started, there is nothing to wait.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by alemagio** [comment]
Is the CI failure related to this?

**Comment by mcollina** [comment]
Likely, keep skipping that test on CI.

**Comment by fox1t** [comment]
I don't understand why waiting makes it pass also on CI. Any clues?

**Comment by alemagio** [comment]
Probably that's why it was skipped in the first place.

**Comment by alemagio** [comment]
Maybe it is not related to the CI failure but I just noticed that the process should exit with code 1, like it does in case of timeout, right?

**Comment by mcollina** [comment]
Let's skip that test, exiting with 0 is good.

**Pull request merged**